extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/namespace/URIExtractionNamespace.java

```diff
@@ -217,13 +217,12 @@ private DelegateParser(
           key,
           input
       ).toString(); // Just in case is long
-      final String val = Preconditions.checkNotNull(
-          inner.get(value),
-          "Value column [%s] missing data in line [%s]",
-          value,
-          input
-      ).toString();
-      return ImmutableMap.<String, String>of(k, val);
+      final Object val = inner.get(value);
+      if (val == null) {
+        // Skip nulls
+        return null;
```

Comment:
Since `Parser.parse()` is not marked as Nullable, maybe saner to return `emptyMap()` here and remove the check in `MapPopulator`?